### PR TITLE
Combine inequality and equality and add SQL examples [ci skip]

### PR DESCRIPTION
**Pull request by tcopeland**
I think these two sections could be combined into one since the "NOT" example is about `!=` as opposed to `NOT IN`.   

But, perhaps folks would prefer to keep them separate... if so, I can rework this to just add examples of the generated SQL to both section.
### Summary

Provide a general description of the code changes in your pull
request... were there any bugs you had fixed? If so, mention them. If
these bugs have open GitHub issues, be sure to tag them here as well,
to keep the conversation linked together.
### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Finally, if your pull request affects documentation or any non-code
changes, guidelines for those changes are [available
here](http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation)

Thanks for contributing to Rails!

### CODE REVIEW

**Comment by tcopeland** [comment]
Oops, sorry, I had forgotten to put `[ci skip]` in the commit msg initially, thus the force push.

**Comment by schneems** [comment]
I think one benifit of having it in it's own section is that you can link to the ID tag directly. Also by having it show up in the table of contents the feature is a bit more discoverable.

I like the SQL addition though. Could you re-work that back into a NOT section?

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -472,12 +472,30 @@ NOTE: Only equality, range and subset checking are possible with Hash conditions
...
+To express inequality, use `where.not`:
+
+```ruby
+Client.where.not(locked: true)
+```
+
+In other words, this query can be generated by calling `where` with no argument, then immediately chain with `not` passing `where` conditions.  This will generate SQL like this:
+
+```sql
+SELECT * FROM clients WHERE (clients.locked != 1)
```
Comment (vipulnsward) [inline guides/source/active_record_querying.md]: 👍

**Comment by tcopeland** [comment]
Ah that's a good point about having it in the index, makes sense.

OK I added a new commit with that change, feel free to squash as you see fit of course.   I left the addition of the example SQL output in "Equality Conditions" section too.

**Comment by maclover7** [comment]
Can you squash your commits?

**Comment by tcopeland** [comment]
You bet!   Squashed and force pushed.

**Comment by vipulnsward** [comment]
🚢

**Comment by schneems** [comment]
Thanks!

**Pull request merged**